toqito/state_props/is_separable.py

```diff
@@ -1,27 +1,71 @@
 """Checks if a quantum state is a separable state."""
 
-from itertools import product
-
 import numpy as np
 from scipy.linalg import orth
 
-from toqito.channel_ops.partial_channel import partial_channel
-from toqito.channels.partial_trace import partial_trace
+from toqito.channel_ops import partial_channel
+from toqito.channels import partial_trace, partial_transpose
 from toqito.channels.realignment import realignment
 from toqito.matrix_props.is_positive_semidefinite import is_positive_semidefinite
 from toqito.matrix_props.trace_norm import trace_norm
 from toqito.perms.swap import swap
 from toqito.perms.swap_operator import swap_operator
 from toqito.state_props.has_symmetric_extension import has_symmetric_extension
 from toqito.state_props.in_separable_ball import in_separable_ball
-from toqito.state_props.is_ppt import is_ppt
 from toqito.state_props.schmidt_rank import schmidt_rank
 from toqito.states.max_entangled import max_entangled
 
+EPS = np.finfo(float).eps
+
 
 def is_separable(state: np.ndarray, dim: None | int | list[int] = None, level: int = 2, tol: float = 1e-8) -> bool:
     r"""Determine if a given state (given as a density matrix) is a separable state :cite:`WikiSepSt`.
 
+    References
+    ==========
+    .. bibliography::
+        :filter: docname in docnames
+
+    :raises ValueError: If dimension is not specified correctly or input is invalid.
+    :param state: The density matrix to check.
+    :param dim: The dimension of the input state, e.g., [dim_A, dim_B].
+    :param level: The level for symmetric extensions.
+    :param tol: Numerical tolerance.
+    :return: True if separable, False if entangled or inconclusive by implemented checks.
+
+    Overview
+    ==========
+    :cite:`Horodecki2009_QuantumEntanglement_Review`
```

Comment:
This appears as blank in the docs build. 

https://toqito--1236.org.readthedocs.build/en/1236/autoapi/state_props/is_separable/index.html